docs/reference/garbage-collection.md

```diff
@@ -158,6 +158,7 @@ You should specify the Uber-jar path instead of `<APPLICATION-JAR-PATH>` and run
 
   ```bash
 spark-submit --class io.treeverse.clients.GarbageCollector \
+  --packages org.apache.hadoop:hadoop-aws:2.7.7 \
```

Comment:
@arielshaqed is this the right hadoop-aws version for all three Spark clients?

Reply:
Not really, of course.  Spark 3 with Hadoop 3 uses some 3.x.y.  E.g. the 312-hadoop3 build is _compiled_ against Hadoop 3.2.1.

Good catch, fixed this version!  But _none_ of the guide really explains Hadoop 2 vs. Hadoop 3 very well.